dist/common/scripts/scylla_ntp_setup

```diff
@@ -40,24 +40,29 @@ if __name__ == '__main__':
                         help='specify subdomain of pool.ntp.org (ex: centos, fedora or amazon)')
     args = parser.parse_args()
 
-    if shutil.which('ntpd'):
-        pkg_uninstall('ntp')
-    if not shutil.which('chronyd'):
-        pkg_install('chrony')
-    if is_debian_variant():
-        confpath = '/etc/chrony/chrony.conf'
-        chronyd = 'chrony.service'
+    # use systemd-timesyncd
+    if systemd_unit.available('systemd-timesyncd'):
```

Comment:
i think you are missing an import of systemd_unit here
and AFAIK it's not installed,so we should install it first

Reply:
systemd_unit is imported at `from scylla_util import *`, and it always usable across scylla setup scripts.